lib/screens/widgets/map.widget.dart

```diff
@@ -76,31 +79,25 @@ class MapWidgetState extends State<MapWidget> {
             (val, marker) => val.longitude > marker.longitude ? val : marker)
         .longitude;
 
-    double centerLat = (minLat + maxLat) / 2;
-    double centerLng = (minLng + maxLng) / 2;
-
-    LatLngBounds bounds =
-        LatLngBounds(LatLng(maxLat, maxLng), LatLng(minLat, minLng));
-
-    CenterZoom zoom = mapController.centerZoomFitBounds(bounds);
```

Comment:
With the updated flutter_map package the map way always grey until the first manual move. I guess this was because this method was called right after build. I've switched it now to set the bounds in the initial `CameraFit`.